binr/radare2/radare2.c

```diff
@@ -886,7 +886,12 @@ int main(int argc, char **argv, char **envp) {
 	}
 
 	if (run_rc) {
+		const char *atstart;
 		radare2_rc (&r);
+		atstart = r_config_get (r.config, "cmd.atstart");
+		if (atstart && *atstart) {
+			r_list_prepend (cmds, (char *)atstart);
```

Comment:
i think may lead into a crash in case cmd.atstart is changed before the cmds are processed

Reply:
probably fixable without using strdup, but superseded by 70b68c47173b65f10675323038b56ec94c2752c0